python/selfie-lib/selfie_lib/EscapeLeadingWhitespace.py

```diff
@@ -0,0 +1,12 @@
+class EscapeLeadingWhitespace:
+    NEVER = "NEVER"
+
+    def __init__(self):
+        self.escape_type = self.NEVER
```

Comment:
This should be an enum like this:

https://github.com/diffplug/selfie-python-wip/blob/f8f51a4d2b1ac22a6df3f61353454b48b38e1c06/python/selfie-lib/selfie_lib/CommentTracker.py#L19-L22

Reply:
Fixed this for an enum.